### PR TITLE
VSIX plugin: No icon for blame

### LINKED ISSUE #4789: VSIX plugin: No icon for blame
I was not able to find an [open](https://github.com/gitextensions/gitextensions/issues?q=is%3Aopen) or [closed](https://github.com/gitextensions/gitextensions/issues?q=is%3Aclosed) issue matching what I'm seeing
Yes

**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
No icon for Blame in VSIX plugin
Some other miss icon too, no obvious icon for them

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
In vsix menu/toolbar

**What is the expected behavior?**

**Environment you encounter the issue:**

- GitExtensions version: all

**Did this work in previous version of GitExtensions (which)?**
No

### PR DESCRIPTION
**Pull request by gerhardol**
resolves #4789

Changes proposed in this pull request:
 - Blame icon
Minor, other commands miss icon still, but this has another use too: See if the old or the new plugin (for instance #4747 ) is used.
 
Screenshots before and after (if PR changes UI):
- 
image

What did I do to test the code and ensure quality:
 - Manual testing

Has been tested on (remove any that don't apply):
 - Windows 10
 - Visual Studio Community 15.6.5

### CODE REVIEW

**Comment by RussKie** [comment]
once #4747 is complete, we will need help from @spdr870 in releasing a new version.

File: GitExtensionsVSIX/GitExtCommandTable.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ internal sealed class PackageGuids
     /// <summary>
     /// Helper class that encapsulates all CommandIDs uses across VS Package.
     /// </summary>
-    internal sealed class PackageIds
+    internal sealed partial class PackageIds
```
Comment (RussKie) [inline GitExtensionsVSIX/GitExtCommandTable.cs]: why has this changed?

Comment (gerhardol) [reply GitExtensionsVSIX/GitExtCommandTable.cs]: Changed by Visual Studio, I let that automatic change pass

Comment (drewnoakes) [reply GitExtensionsVSIX/GitExtCommandTable.cs]: Looks like a newer version of the tooling, from an earlier change in this file. Harmless enough.

Comment (RussKie) [reply GitExtensionsVSIX/GitExtCommandTable.cs]: Nw, just checking.
I noticed that VS is modifying resources in the GitUI and after those automatic changes resources not longer build.

**Pull request merged**